.github/workflows/backport-assistant.yml

```diff
@@ -16,12 +16,16 @@ jobs:
   backport:
     if: github.event.pull_request.merged
     runs-on: ubuntu-latest
-    container: hashicorpdev/backport-assistant:0.3.0
+    container: hashicorpdev/backport-assistant:0.3.5
     steps:
       - name: Run Backport Assistant for release branches
         run: |
           backport-assistant backport -merge-method=squash -gh-automerge
         env:
           BACKPORT_LABEL_REGEXP: "backport/(?P<target>\\d+\\.\\d+)"
           BACKPORT_TARGET_TEMPLATE: "release/{{.target}}.x"
+          # This forces the backport assistant to backport the merged commit
+          # instead of each commit individually. The environment variable
+          # just needs to exist for this to happen.
+          BACKPORT_MERGE_COMMIT: false
```

Comment:
Should we put in a value of `true` if the value simply needs to exist? Maybe I'm misunderstanding, but if the existence of `BACKPORT_MERGE_COMMIT` means it's true, then explicitly setting the value of false is super misleading.

Reply:
It's still super confusing in the description lol